src/working-here/pay-pension-and-benefits.md

```diff
@@ -256,7 +250,7 @@ point. The sooner we know, the sooner we can support you. We will never use the
 disclosure of a disability or long term health condition against you.
 
 We’ll organise a meeting (or a series of meetings if needed) between you, your
-line manager, and the people manager where we will discuss any difficulties you
+line manager, and the people team where we will discuss any difficulties you
```

Comment:
@SamJCoop676 do you want to capitalise `people` as `People` in line with the other instances?

Reply:
yes please